### ISSUE TITLE
Info: Amazing Demo

### ISSUE THREAD

**Issue by ccoupe** [shoes/shoes3#445]
This is just a note. This [Apple presentation](ttps://developer.apple.com/videos/play/wwdc2019/204/) is very interesting. Notice how the "new" layout uses stacks and flows - just like Shoes.

The ability to show the results in real time is similar to the shoes manual "Run This". The ability to edit the properties in the other window and modify the code? We don't have that. @BackOrder once threatened to write such a create.

**Comment by dredknight** [comment]
dang... it is kind of they use rebranded Shoes...

**Comment by ccoupe** [comment]
_why's HacketyHack's editer is  interesting (a little two garish for me) and a bit buggy  with newer Shoes but it would be a place to start.  Just needs to replace hpricot with nokogiri to try it out (or install hpricot).

BTW, I'm working on getting the old Win32 native code working again.  Much of shoes works - it's just missing much of the stuff added/fixed since  2014. Why? I'm really tired of fighting with Gtk on windows. A very real bonus is it's Fast to load. Looks better too.

**Comment by dredknight** [comment]
Btw "The ability to edit the properties in the other window and modify the code? " you mean better way to allow the user to manage one window from another? currently you have to make certain assignments to keep things organized and there are issues in some calls as far as I remember.

What is this code "BTW, I'm working on getting the old Win32 native code working again" ?
What is the future of GTK for shoes? you are going back to the old GTK or something else I cannot comprehend.

**Comment by ccoupe** [comment]
>  user to manage one window from another? currently you have to make certain assignments to keep things organized and there are issues in some calls as far as I remember.

Yes, Shoes makes it difficult to do causally, but it's  a top-down planning anddesign problem first and I don't work that way for very long.

> What is the future of GTK for shoes?

Assuming I get the Windows native widget  code working then we wouldn't need Gtk on Windows. Shoes would still be Shoes. There is an English pun to remember: "Assume" makes an ass out of you and me. 

I may never get the _old_ windows code working to our satisfaction. If so, we still have the Gtk version for Windows.  There a lots of ways to build Shoes for Windows. One of them will always work. Currently it's the Gtk version. If they both work, one will be 'better'.
